src/scripts/plc.ts

```diff
@@ -162,5 +180,5 @@ export async function plcClearLabeler(options: PlcClearLabelerOptions) {
  */
 export async function plcRequestToken(credentials: LoginCredentials): Promise<void> {
 	const { agent } = await loginAgent(credentials);
-	await agent.call("com.atproto.identity.requestPlcOperationSignature", {});
+	await agent.post("com.atproto.identity.requestPlcOperationSignature", { as: null });
```

Comment:
I'm not 100% sure if this is the correct usage here?

Reply:
Looks funny but it does appear to be correct